### ISSUE TITLE
Pure JS client library.

### ISSUE THREAD

**Issue by SukantGujar** [colinskow/ng-superlogin#8]
Hi,

I am planning to try out Superlogin for my React based application. However, the client library is only available in Angular flavor. I was wondering if you would be interested in providing a pure JS version.

Thanks and regards,
Sukant

**Comment by micky2be** [comment]
Hi @SukantGujar 
I needed a pure JS solution as well for my ReactJS app. I just released the client on NPM.
https://www.npmjs.com/package/superlogin-client

Feel free to try it out.

**Comment by SukantGujar** [comment]
Wow thanks!
On Mar 30, 2016 1:14 PM, Micky Brunetti notifications@github.com wrote:Hi @SukantGujar 
I needed a pure JS solution as well for my ReactJS app. I just released the client on NPM.
https://www.npmjs.com/package/superlogin-client

Feel free to try it out.

**Comment by colinskow** [comment]
@micky2be that is awesome!!! Thank you very much. I'm glad so many people are getting involved in the SuperLogin community.

**Comment by colinskow** [comment]
TODO: reference `superlogin-client` in readme files.

**Comment by SukantGujar** [comment]
Yeah :), but I must admit I super new to PouchDB and superlogin. Would you mind terribly if I bug you for advise in case I come across a hurdle?
Regards,
Sukant
On Apr 3, 2016 11:42 AM, Colin Skow notifications@github.com wrote:@micky2be that is awesome!!! Thank you very much. I'm glad so many people are getting involved in the SuperLogin community.

**Comment by micky2be** [comment]
I don't mind much. But @colinskow is probably the goto guy.

**Comment by colinskow** [comment]
Go to http://slack.pouchdb.com or join #pouchdb on IRC. On PouchDB Slack #SuperLogin has a special channel.
